### DISCUSSION TITLE
Add location to engineers

### DISCUSSION THREAD

**Discussion by ternite** [iaincollins/icarus#9] [Ideas & Suggestions]
Hi,
it's quite useful ICARUS shows which engineers can be used.

I assume it only shows those I unlocked? (does it?)

My suggestion is to add the system name behind the engineer's name for easier reference.

Also, if only unlocked engineers are shown, how about adding greyed out engineers that have not already been unlocked?

Best regards

**Comment by iaincollins** [comment]
Hi there,

The Blueprint view currently shows all engineers (locked and unlocked).

The plan is to show an icon/text to indicate which engineers are locked/unlocked, the name of the system they are in and the distance to that system from your current location. This is likely to be implemented fairly soon.

I've also considered a panel that lists each engineer and the various engineering Blueprints and Experimental effects they can apply and include information about how to unlock each engineer.

**Comment by iaincollins** [comment]
Quick update to say that in v0.10.0, released yesterday, the location and current distance for each engineer is displayed on the Blueprints screen.

Additional work (e.g. locked/unlocked status for each engineer, dedicated pages for each engineer with a list of things they can help with, how to unlock them, etc) is still planned for the future.

![v0-11-1-eng](https://user-images.githubusercontent.com/595695/164385761-aaf78779-4dd5-42b0-beab-4781915fc7da.png)

**Reply by ternite** [reply]
Unlock status is now present, too. I like it, thanks!

![Screenshot 2022-05-03 101928](https://user-images.githubusercontent.com/7860777/166423308-763406a1-6ec8-460e-aa58-e963d5258f95.png)
